site/components/buttons/DeleteDisruptionButton.tsx

```diff
@@ -23,7 +25,7 @@ const DeleteDisruptionButton = ({
             {showDeleteModal && (
                 <DeleteConfirmationPopup
                     entityName={`the disruption`}
```

Comment:
Pop up currently shows "Are you sure you wish to delete the disruption?"

Also need to make entityName conditional to pass through "the template" if user in is template flow

Reply:
done